sig/mutex_m.rbs

```diff
@@ -0,0 +1,99 @@
+# # mutex_m.rb
+#
+# When 'mutex_m' is required, any object that extends or includes Mutex_m will
+# be treated like a Mutex.
+#
+# Start by requiring the standard library Mutex_m:
+#
+#     require "mutex_m.rb"
```

Comment:
Dumb question: is the .rb suffix required here?

Reply:
Yeah! 💡 We don't need the suffix.
This is a copy of the original RDoc from `lib/mutex_m.rb`, so we won't fix it here.

Reply:
My understanding (right now) is that the ruby/ruby no longer has the file.

I opened a PR in this repo, to make the docs change.

Reply:
Regenerated the RBS docs with updated RDoc content. 👍